.golangci.yml

```diff
@@ -62,7 +71,7 @@ linters-settings:
     check-blank: false
   govet:
     # report about shadowed variables
-    check-shadowing: true
+    check-shadowing: false
```

Comment:
this lint is to check shadow declaration of variable. Lot of times we re-declare `err` by using `:=` operator.

Let me know if you want to fix it, I can make the changes.

```
time "/Users/tammach/go/bin/golangci-lint" run
pkg/gitops/flux/installer.go:166:5         govet  shadow: declaration of "err" shadows declaration at line 133
pkg/gitops/flux/installer.go:172:6         govet  shadow: declaration of "err" shadows declaration at line 133
pkg/gitops/flux/installer.go:182:5         govet  shadow: declaration of "err" shadows declaration at line 133
pkg/gitops/profile.go:52:5                 govet  shadow: declaration of "err" shadows declaration at line 46
pkg/addons/vpc_controller.go:98:7          govet  shadow: declaration of "err" shadows declaration at line 87
pkg/nodebootstrap/userdata.go:104:5        govet  shadow: declaration of "err" shadows declaration at line 96
pkg/kubernetes/client.go:313:6             govet  shadow: declaration of "err" shadows declaration at line 308
pkg/kubernetes/client.go:357:6             govet  shadow: declaration of "err" shadows declaration at line 351
pkg/kubernetes/client.go:395:6             govet  shadow: declaration of "err" shadows declaration at line 351
pkg/eks/api.go:206:5                       govet  shadow: declaration of "err" shadows declaration at line 196
pkg/ctl/create/cluster.go:112:5            govet  shadow: declaration of "err" shadows declaration at line 94
pkg/ctl/create/cluster.go:116:5            govet  shadow: declaration of "err" shadows declaration at line 94
pkg/ctl/create/cluster.go:134:7            govet  shadow: declaration of "err" shadows declaration at line 94
pkg/ctl/create/cluster.go:375:7            govet  shadow: declaration of "err" shadows declaration at line 346
pkg/ctl/create/cluster.go:381:7            govet  shadow: declaration of "err" shadows declaration at line 346
pkg/ctl/create/cluster.go:384:7            govet  shadow: declaration of "err" shadows declaration at line 346
pkg/ctl/create/fargate.go:65:5             govet  shadow: declaration of "err" shadows declaration at line 61
pkg/ctl/create/fargate.go:69:9             govet  shadow: declaration of "err" shadows declaration at line 61
pkg/ctl/create/fargate.go:81:5             govet  shadow: declaration of "err" shadows declaration at line 61
pkg/ctl/create/iamidentitymapping.go:56:5  govet  shadow: declaration of "err" shadows declaration at line 51
pkg/ctl/create/iamidentitymapping.go:68:5  govet  shadow: declaration of "err" shadows declaration at line 51
pkg/ctl/create/iamidentitymapping.go:76:9  govet  shadow: declaration of "err" shadows declaration at line 51
pkg/ctl/create/iamserviceaccount.go:71:5   govet  shadow: declaration of "err" shadows declaration at line 65
pkg/ctl/create/iamserviceaccount.go:75:9   govet  shadow: declaration of "err" shadows declaration at line 65
pkg/ctl/create/nodegroup.go:86:5           govet  shadow: declaration of "err" shadows declaration at line 80
pkg/ctl/create/nodegroup.go:90:9           govet  shadow: declaration of "err" shadows declaration at line 80
pkg/ctl/create/nodegroup.go:94:5           govet  shadow: declaration of "err" shadows declaration at line 80
pkg/ctl/delete/cluster.go:92:5             govet  shadow: declaration of "err" shadows declaration at line 86
pkg/ctl/delete/cluster.go:97:5             govet  shadow: declaration of "err" shadows declaration at line 86
pkg/ctl/delete/cluster.go:101:9            govet  shadow: declaration of "err" shadows declaration at line 86
pkg/ctl/delete/fargate.go:58:5             govet  shadow: declaration of "err" shadows declaration at line 54
pkg/ctl/delete/iamidentitymapping.go:53:5  govet  shadow: declaration of "err" shadows declaration at line 47
pkg/ctl/delete/iamidentitymapping.go:64:9  govet  shadow: declaration of "err" shadows declaration at line 47
pkg/ctl/delete/iamidentitymapping.go:76:5  govet  shadow: declaration of "err" shadows declaration at line 47
pkg/ctl/delete/iamserviceaccount.go:71:5   govet  shadow: declaration of "err" shadows declaration at line 65
pkg/ctl/delete/nodegroup.go:62:5           govet  shadow: declaration of "err" shadows declaration at line 56
pkg/ctl/delete/nodegroup.go:66:9           govet  shadow: declaration of "err" shadows declaration at line 56
pkg/ctl/drain/nodegroup.go:58:5            govet  shadow: declaration of "err" shadows declaration at line 52
pkg/ctl/drain/nodegroup.go:62:9            govet  shadow: declaration of "err" shadows declaration at line 52
pkg/ctl/enable/profile.go:87:5             govet  shadow: declaration of "err" shadows declaration at line 83
pkg/ctl/enable/repo.go:54:5                govet  shadow: declaration of "err" shadows declaration at line 50
pkg/ctl/enable/utils.go:18:5               govet  shadow: declaration of "err" shadows declaration at line 14
pkg/ctl/get/fargate.go:64:5                govet  shadow: declaration of "err" shadows declaration at line 60
pkg/ctl/get/iamidentitymapping.go:57:5     govet  shadow: declaration of "err" shadows declaration at line 52
pkg/ctl/get/iamserviceaccount.go:63:5      govet  shadow: declaration of "err" shadows declaration at line 58
pkg/ctl/get/iamserviceaccount.go:67:9      govet  shadow: declaration of "err" shadows declaration at line 58
pkg/ctl/get/nodegroup.go:69:5              govet  shadow: declaration of "err" shadows declaration at line 64
pkg/ctl/scale/nodegroup.go:68:5            govet  shadow: declaration of "err" shadows declaration at line 63
pkg/ctl/update/cluster.go:63:5             govet  shadow: declaration of "err" shadows declaration at line 57
pkg/ctl/update/cluster.go:67:9             govet  shadow: declaration of "err" shadows declaration at line 57
```

Reply:
Right, I wonder how this was working before :thinking: If there is a way to only allow shadowing `err` and `ctx` that would be nice. Otherwise let's leave it like that for now.

Reply:
There is no way that I am aware of. I think i just leave it for now :(